frontend/src/services/user.ts

```diff
@@ -1,39 +1,22 @@
 import {User, UserCredentials, UserType} from "../types/user";
+import httpClient from "./httpClient";
+import {HttpPaths} from "../common/constansts/paths";
+import _ from "lodash";
 
 export const getUserService = async (userId: number, userType: UserType): Promise<User> => {
     if (userType === UserType.Customer) {
-        return {
-            id: userId,
-            email: "user@gmail.com",
-            name: "User",
-            createdAt: new Date(),
-            updatedAt: new Date(),
-        }
-    }
-    return {
-        id: userId,
-        email: "usermerchant@gmail.com",
-        name: "UserMerchant",
-        createdAt: new Date(),
-        updatedAt: new Date(),
+        const response = await httpClient.get<User>(`${HttpPaths.CUSTOMER}/${userId}`);
+        return _.get(response, "data");
     }
+    const response = await httpClient.get<User>(`${HttpPaths.MERCHANT}/${userId}`);
+    return _.get(response, "data");
 }
 
-export const signUpService = async (userCredentials: UserCredentials, userType: UserType): Promise<User> => {
+export const signUpService = async (userCredentials: UserCredentials, userType: UserType): Promise<User | null> => {
     if (userType === UserType.Customer) {
-        return {
-            id: 1,
-            email: "user@gmail.com",
-            name: "User",
-            createdAt: new Date(),
-            updatedAt: new Date(),
-        }
-    }
-    return {
-        id: 1,
-        email: "usermerchant@gmail.com",
-        name: "UserMerchant",
-        createdAt: new Date(),
-        updatedAt: new Date(),
+        const response = await httpClient.post<User>(HttpPaths.CUSTOMER + HttpPaths.SIGN_UP_USER, userCredentials);
```

Comment:
why do you use here string + string, but above `${string}/${stirng}` ?

Reply:
in get method i provide only request param(userId) without body, in post method different situation, use body without any param in url